spec/controllers/orders_controller_spec.rb

```diff
@@ -683,46 +680,71 @@ def deliver; end
           @submitted_date = 2.days.ago.change(hour: 14, min: 27)
         end
 
-        it "sets the order_detail states to completed because it's in the past" do
-          do_request
-          assigns[:order].order_details.all? { |od| expect(od.state).to eq("complete") }
-        end
+        context "for a reservation-only instrument" do
+          before :each do
+            @instrument = FactoryGirl.create(:setup_instrument,
```

Comment:
We've started the transitioning this file, so it would be nice to switch what we can (without going too overboard, of course). You'll see at the top of the file we have things like (simplified):

```
let(:facility) { create(:facility) }

before { @authable = facility }
```

I think it would be nice to at least use the `facility` from `let` here rather than `@authable`.

It might also be worth considering trying to switch to an `instrument` let (might involve a little changing to `setup_past_reservation`). 

On the other hand, this file is a mess, and is probably more effort than it's worth to clean up right now.